glide-core/src/request_type.rs

```diff
@@ -9,224 +9,421 @@ use crate::command_request::RequestType as ProtobufRequestType;
+    MemoryMallocStats = 1140,
+    MemoryPurge = 1141,
+    MemoryStats = 1142,
+    MemoryUsage = 1143,
+    ModuleList = 1144,
+    ModuleLoad = 1145,
+    ModuleLoadEx = 1146,
+    ModuleUnload = 1147,
+    Monitor = 1148,
+    PSync = 1149,
+    ReplConf = 1150,
+    ReplicaOf = 1151,
+    RestoreAsking = 1152,
+    Role = 1153,
+    Save = 1154,
+    ShutDown = 1155,
+    SlaveOf = 1156,
+    SlowLogGet = 1157,
+    SlowLogLen = 1158,
+    SlowLogReset = 1159,
+    SwapDb = 1160,
+    Sync = 1161,
+    Time = 1162,
+
+    //// Set commands
+    SAdd = 1201,
+    SCard = 1202,
+    SDiff = 1203,
+    SDiffStore = 1204,
+    SInter = 1205,
+    SInterCard = 1206,
+    SInterStore = 1207,
+    SIsMember = 1208,
+    SMembers = 1209,
+    SMIsMember = 1210,
+    SMove = 1211,
+    SPop = 1212,
+    SRandMember = 1213,
+    SRem = 1214,
+    SScan = 1215,
+    SUnion = 1216,
+    SUnionStore = 1217,
+
+    //// Sorted set commands
+    BZMPop = 1301,
+    BZPopMax = 1302,
+    BZPopMin = 1303,
+    ZAdd = 1304,
+    ZCard = 1305,
+    ZCount = 1306,
+    ZDiff = 1307,
+    ZDiffStore = 1308,
+    ZIncrBy = 1309,
+    ZInter = 1310,
+    ZInterCard = 1311,
+    ZInterStore = 1312,
+    ZLexCount = 1313,
+    ZMPop = 1314,
+    ZMScore = 1315,
+    ZPopMax = 1316,
+    ZPopMin = 1317,
+    ZRandMember = 1318,
+    ZRange = 1319,
+    ZRangeByLex = 1320,
+    ZRangeByScore = 1321,
+    ZRangeStore = 1322,
+    ZRank = 1323,
+    ZRem = 1324,
+    ZRemRangeByLex = 1325,
+    ZRemRangeByRank = 1326,
+    ZRemRangeByScore = 1327,
+    ZRevRange = 1328,
+    ZRevRangeByLex = 1329,
+    ZRevRangeByScore = 1330,
+    ZRevRank = 1331,
+    ZScan = 1332,
+    ZScore = 1333,
+    ZUnion = 1334,
+    ZUnionStore = 1335,
+
+    //// Stream commands
+    XAck = 1401,
+    XAdd = 1402,
+    XAutoClaim = 1403,
+    XClaim = 1404,
+    XDel = 1405,
+    XGroupCreate = 1406,
+    XGroupCreateConsumer = 1407,
+    XGroupDelConsumer = 1408,
+    XGroupDestroy = 1409,
+    XGroupSetId = 1410,
+    XInfoConsumers = 1411,
+    XInfoGroups = 1412,
+    XInfoStream = 1413,
+    XLen = 1414,
+    XPending = 1415,
+    XRange = 1416,
+    XRead = 1417,
+    XReadGroup = 1418,
+    XRevRange = 1419,
+    XSetId = 1420,
+    XTrim = 1421,
+
+    //// String commands
+    Append = 1501,
+    Decr = 1502,
+    DecrBy = 1503,
+    Get = 1504,
+    GetDel = 1505,
+    GetEx = 1506,
+    GetRange = 1507,
+    GetSet = 1508, // deprecated in 6.2.0
+    Incr = 1509,
+    IncrBy = 1510,
+    IncrByFloat = 1511,
+    LCS = 1512,
+    MGet = 1513,
+    MSet = 1514,
+    MSetNX = 1515,
+    PSetEx = 1516, // deprecated in 2.6.12
+    Set = 1517,
+    SetEx = 1518, // deprecated in 2.6.12
+    SetNX = 1519, // deprecated in 2.6.12
+    SetRange = 1520,
+    Strlen = 1521,
+    Substr = 1522,
+
+    //// Transaction commands
+    Discard = 1601,
+    Exec = 1602,
+    Multi = 1603,
+    UnWatch = 1604,
+    Watch = 1605,
+
+    //// JSON commands
+    JsonArrAppend = 2001,
```

Comment:
I think we should split this PR into two: 
1. clean up all the existing commands
2. add the JSON and VSS commands